app/assets/stylesheets/article-show.scss

```diff
@@ -1508,10 +1508,24 @@ article {
   padding: 50% 30 50% 80;
 }
 
+.youtube-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+}
+
+.yt-info {
+  opacity: 0;
+  width: .0001em;
+  height: 0;
+}
```

Comment:
@dreardon1021  I'm assuming this styling allows the information to be rendered on the page but keeps it invisible to the user.